api/app/main.py

```diff
@@ -32,6 +32,11 @@ async def read_item() -> Any:
     return TodoListCreateResponse(id=await todo_store.create_todo_list())
 
 
+@app.patch("/todolist/{list_id}/{new_name}")
```

Comment:
I think I would prefer to use either a request body here or a query parameter (vs a path parameter)

Generally path parameters are used to specify the _location_ of a resource, rather than indicating a new value

Reply:
resolved in commit: 7e7b12a72e88d3199dcb349426ebe6a9ab92d06e